service/controller/resource/monitoring/prometheus/resource.go

```diff
@@ -102,6 +104,12 @@ func toPrometheus(ctx context.Context, v interface{}, config Config) (metav1.Obj
 		return nil, microerror.Mask(err)
 	}
 
+	// TODO: use the return value to compute request memory for Prometheus.
+	_, err = getClusterNodeCount(ctx, config.K8sClient, cluster)
```

Comment:
Could we not name this countClusterNodes?

Reply:
Yes sure